.gitignore

```diff
@@ -1,42 +1,176 @@
-dist
-*classes
-target/
-build/
-.gradle/
+# Compiled from https://github.com/github/gitignore
+# With Kafka Specific alterations
+
+# Covers Linux OS
+*~
+## KDE directory preferences
+.directory
+## Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+# Covers Windows OS
+## Windows image file caches
+Thumbs.db
+ehthumbs.db
+## Folder config file
+Desktop.ini
+## Recycle Bin used on file shares
+$RECYCLE.BIN/
+## Windows Installer files
+*.cab
+*.msi
+*.msm
+*.msp
+## Windows shortcuts
+*.lnk
+
+# Covers Mac OSX
+.DS_Store
+.AppleDouble
+.LSOverride
+## Thumbnails
+._*
+## Files that might appear in the root of a volume
```

Comment:
This seems like another set that don't make sense here. How would you get these into your Kafka checkout?